### ISSUE TITLE
Report event with new report / new contribution

### ISSUE THREAD

**Issue by loma-one** [friendica/friendica#13736]
### Is the feature request related to a problem? Please describe.

If a new report (report contribution) is generated by a user of the same instance or if a report reaches the instance, it would be great if an event were generated via the internal notification system. 
At present, it is necessary to check manually and cyclically whether such a report has been received.

**Comment by shiribailem** [comment]
Yeah, getting really frustrated with this behavior when I realize I haven't checked in a few days and find serious things to deal with...

And in the meantime can anyone tell me maybe a database query or something to pull reports so I can just jury-rig a script to notify me in the meantime? Or maybe some API method?

**Comment by annando** [comment]
@shiribailem you could have a look at the table `report`.
